redfish-core/lib/log_services.hpp

```diff
@@ -51,6 +53,32 @@ constexpr char const* crashdumpOnDemandInterface =
 constexpr char const* crashdumpTelemetryInterface =
     "com.intel.crashdump.Telemetry";
 
+constexpr auto objMapperSerName = "xyz.openbmc_project.ObjectMapper";
```

Comment:
We don't do this in any other mappers calls, lets not do this here

Reply:
Can I know the reason why we don't want to do it? FYI, I am using this constexpr in many places so, if we want to change in future (agree very rare) it will be easy to change in one place.

Any advantage if we hardcoding in all places?

Reply:
It is easier to verify the call. 
I want to match upstream as much as possible with this match